src/include/sof/audio/audio_stream.h

```diff
@@ -46,17 +46,24 @@ struct sof_audio_stream_params {
 	uint32_t rate;		/**< Number of data frames per second [Hz] */
 	uint16_t channels;	/**< Number of samples in each frame */
 
-	/** alignment limit of stream copy, this value indicates how many
-	 * integer frames which can meet both byte align and frame align
-	 * requirement. it should be set in component prepare or param functions
+	/**
+	 * align_frame_cnt indicates minimum frames can meet both bytes align and
```

Comment:
not a native English speaker, but maybe "minimum number of frames, that satisfies both byte and frame alignment requirements."